source/user-manual/manager/configuring-cluster/advanced-settings.rst

```diff
@@ -1,27 +1,723 @@
+
+Some of its key features are:
+
+* Add/remove new servers to the Wazuh backend (1514/tcp) when detecting changes on the Wazuh cluster (e.g. new workers connected).
+* Balance excess agents per node when adding new servers to the Wazuh backend.
+* Balance agents when detecting an imbalance that exceeds the given tolerance.
+
+.. thumbnail:: /images/manual/cluster/haproxy-helper-architecture.png
+    :title: HAProxy helper architecture
+    :alt: HAProxy helper architecture
+    :align: center
+    :width: 80%
+
+
+The helper runs in an independent thread, that initiates with the ``wazuh-cluster`` daemon, and completes the next flow:
+
+.. thumbnail:: /images/manual/cluster/haproxy-helper-flow.png
+    :title: HAProxy helper flow
+    :alt: HAProxy helper flow
+    :align: center
+    :width: 80%
+
+
+How to enable it
+----------------
+
+.. note::
+    The recommended version of HAProxy is the 2.8 LTS.
+
+To use this feature is required to have a :ref:`HAProxy <haproxy_installation>` instance balancing the cluster using the **least connections** algorithm.
+
+Dataplane API configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    The Dataplane API is used by the helper to communicate with HAProxy and update the configuration according to the changes in the Wazuh cluster.
+
+    This is the basic configuration:
+
+    .. code-block:: yaml
+
+        dataplaneapi:
+            host: 0.0.0.0
+            port: 5555
+            transaction:
+                transaction_dir: /tmp/haproxy
+            user:
+            - insecure: true
+              password: <DATAPLANE_PASSWORD>
+              name: <DATAPLANE_USER>
+        haproxy:
+            config_file: /etc/haproxy/haproxy.cfg
+            haproxy_bin: /usr/sbin/haproxy
+            reload:
+                reload_delay: 5
+                reload_cmd: service haproxy reload
+                restart_cmd: service haproxy restart
 
     .. note::
 
-        We recommend using the **load balancer** option. This way, the agents will be able to report to the nodes in a distributed way and it will be the load balancer who assigns which worker they report to. Using this option, we can better distribute the load, and in case of a fall in some worker node, its agents will **reconnect** to another one.
+        Is needed to replace ``<DATAPLANE_USER>`` and ``<DATAPLANE_PASSWORD>`` with the chosen user and password.
+
+    To enable it will depend on the :ref:`installation method <haproxy_installation>`.
+
+    .. warning::
+
+        For the correct operation of the helper, there must not be a frontend, with the port **1514**, in the ``haproxy.cfg`` file beforehand.
+
+    .. tabs::
+        .. group-tab:: Packages
+
+            1. Download the binary file for the installed HAProxy version
+
+            .. code-block:: console
+
+                # curl -sL https://github.com/haproxytech/dataplaneapi/releases/download/v2.8.X/dataplaneapi_2.8.X_linux_x86_64.tar.gz | tar xz && cp dataplaneapi /usr/local/bin/
```

Comment:
```suggestion
                # curl -sL https://github.com/haproxytech/dataplaneapi/releases/download/v2.8.7/dataplaneapi_2.8.7_linux_x86_64.tar.gz | tar xz && cp dataplaneapi /usr/local/bin/
```
One last comment on my side: are these Xs intentional? If we don't want to include the patch in case they update it (which seems fine to me), we should at least tell the user where they can find the latest one: 
- https://github.com/haproxytech/dataplaneapi/releases/

Maybe something like 
```
1. Download the binary file for the installed HAProxy version. You can find the available versions [here](https://github.com/haproxytech/dataplaneapi/releases/). 
```